### PR TITLE
fix add new cards

### PR DESCRIPTION
**Pull request by underdoeg**
Sorry my last PR introduced a major bug that prevented users from adding new projects, clients and tasks.

### CODE REVIEW

**Comment by underdoeg** [comment]
hmm now I am confused, why does this list 17 commits, it should only be the last 3..

**Comment by overshard** [comment]
@underdoeg this PR has some odd issues in it, there seems to be random text copied into the code

File: client/static_src/components/selector.vue [hunk 1]
```diff
@@ -1,3 +1,27 @@
+
```
Comment (overshard) [inline client/static_src/components/selector.vue]: Accidentally pasted in?

File: client/static_src/components/selector.vue [hunk 2]
```diff
@@ -184,25 +206,36 @@
...
     max-height: 200px;
     overflow-y: scroll;
     background-color: #fff;
   }
-
   .list-group-item {
     padding: .25rem .5rem;
   }
 </style>
+
```
Comment (overshard) [inline client/static_src/components/selector.vue]: Accidentally pasted in?

Comment (underdoeg) [reply client/static_src/components/selector.vue]: looks like it. I'll clean it up...

**Pull request closed without merging**